### PR TITLE
feat(aws): add aws-nuke and cloud-nuke

### PR DESCRIPTION
**Pull request by dudymas**
## what
* add aws-nuke
* add cloud-nuke

## why
* both provide opinionated ways for clearing aws accounts
* cloud-nuke can be appropriate for a one-off account that does not serve a tier1 (ie prod) purpose
* aws-nuke can be appropriate for tier1 accounts where multiple accounts are expected for a root/superadmin

## references
* https://github.com/gruntwork-io/cloud-nuke
* https://github.com/rebuy-de/aws-nuke

### CODE REVIEW

File: vendor/cloud-nuke/Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Package details
+export VENDOR ?= gruntwork-io
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/$(PACKAGE_NAME)_$(OS)_$(ARCH)
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+#export APKBUILD_DEPENDS += # add dependencies here
```
Comment (nitrocode) [inline vendor/cloud-nuke/Makefile:5]:
```suggestion
```

**Review by nitrocode — CHANGES_REQUESTED** [review]
This won't work as is because the github actions have not been generated. 

Please look at commands in this pr.

https://github.com/cloudposse/packages/pull/2430

File: vendor/aws-nuke/Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export VENDOR ?= rebuy-de
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+export PACKAGE_REPO_NAME = aws-nuke
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/$(PACKAGE_REPO_NAME)-v$(PACKAGE_VERSION)-$(OS)-$(ARCH).tar.gz
+#export APKBUILD_DEPENDS += # add dependencies here
```
Comment (nitrocode) [inline vendor/aws-nuke/Makefile:5]:
```suggestion
```

Comment (dudymas) [reply vendor/aws-nuke/Makefile]: fixed

**Comment by dudymas** [comment]
@nitrocode thanks! I think I figured out what was unhappy. I'll mention in channel once this is all green

**Review by osterman — APPROVED** [review]

**Review by nitrocode — APPROVED** [review]

**Pull request merged**